doc/twisted-support.rst

```diff
@@ -1,13 +1,51 @@
 Twisted support
 ===============
 
-testtools provides support for running Twisted tests – tests that return a
-Deferred_ and rely on the Twisted reactor.
+testtools provides support for testing Twisted code.
+
+
+Matching Deferreds
+------------------
+
+testtools provides support for making assertions about synchronous
+:py:class:`~twisted.internet.defer.Deferred`\s.
+
+A "synchronous" :py:class:`~twisted.internet.defer.Deferred` is one that does
+not need the reactor or any other asynchronous process in order to fire.
+
+Normal application code can't know when a
+:py:class:`~twisted.internet.defer.Deferred` is going to fire, because that is
+generally left up to the reactor. Well-written unit tests provide fake
+reactors, or don't use the reactor at all, so that
+:py:class:`~twisted.internet.defer.Deferred`\s fire synchronously.
```

Comment:
I guess you probably checked how this renders but I thought you had to do "...Deferred\`\ s"` to get this to work. (note the space after the backslash).